Configuration/StandardSequences/python/SimL1EmulatorRepack_uGT_cff.py

```diff
@@ -30,6 +30,7 @@
 simGtStage2Digis.JetInputTag        = "unpackGtStage2:Jet"
 simGtStage2Digis.EtSumInputTag      = "unpackGtStage2:EtSum"
 simGtStage2Digis.EtSumZdcInputTag   = "unpackGtStage2:EtSumZDC"
+simGtStage2Digis.CICADAScoreInputTag = "unpackGtStage2:CICADAScore"
```

Comment:
@aloeliger 

when trying to run the L1T emulation in `CMSSW_14_2_X` I got:

```
----- Begin Fatal Exception 04-Nov-2024 15:06:01 CET-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named hlt.py
Exception Message:
 unknown python problem occurred.
TypeError: CICADAScoreInputTag does not already exist, so it can only be set to a CMS python configuration type

At:
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week0/el9_amd64_gcc12/cms/cmssw/CMSSW_14_2_X_2024-11-03-0000/src/FWCore/ParameterSet/python/Mixins.py(319): __raiseBadSetAttr
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week0/el9_amd64_gcc12/cms/cmssw/CMSSW_14_2_X_2024-11-03-0000/src/FWCore/ParameterSet/python/Mixins.py(244): __addParameter
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week0/el9_amd64_gcc12/cms/cmssw/CMSSW_14_2_X_2024-11-03-0000/src/FWCore/ParameterSet/python/Mixins.py(285): __setattr__
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week0/el9_amd64_gcc12/cms/cmssw/CMSSW_14_2_X_2024-11-03-0000/src/Configuration/StandardSequences/python/SimL1EmulatorRepack_uGT_cff.py(33): <module>
  <frozen importlib._bootstrap>(228): _call_with_frames_removed
  <frozen importlib._bootstrap_external>(850): exec_module
  <frozen importlib._bootstrap>(695): _load_unlocked
  <frozen importlib._bootstrap>(986): _find_and_load_unlocked
  <frozen importlib._bootstrap>(1007): _find_and_load
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week0/el9_amd64_gcc12/cms/cmssw/CMSSW_14_2_X_2024-11-03-0000/src/FWCore/ParameterSet/python/Config.py(762): load
  /cvmfs/cms-ib.cern.ch/sw/x86_64/week0/el9_amd64_gcc12/cms/cmssw/CMSSW_14_2_X_2024-11-03-0000/src/HLTrigger/Configuration/python/CustomConfigs.py(178): L1REPACK
  hlt.py(31207): <module>

----- End Fatal Exception -------------------------------------------------
```

I fixed it at https://github.com/cms-sw/cmssw/pull/46597